docs/learn/ticks-and-update-rates.md

```diff
@@ -1,58 +1,39 @@
+## Tick rate
 
+Tick rate, also known as the simulation rate, is a measure of how frequently the server updates the game state. At the beginning of a tick, the server starts to process the data it's received from clients since the last tick and runs the necessary simulations to update the game state. The updated game state is then sent back to the clients. The faster the server finishes a tick, the earlier clients can receive the updated game state (subject to the client's [update rate](#update-rate)).
 
-## Interpolation Period
-
-A period of time it takes to smoothly transition from old state to the new state.
-
-## Packet Sending Period 
-
-The time the server waits between sending packets to clients.
-
-## Latency and Lag
+![Tick rate](/img/tick_rate.png)
 
-See [Latency](lagandpacketloss.md#latency)
+Tick rate is measured in hertz (Hz). For example, a tick rate of 60 Hz means the server updates the game state sixty times a second, whereas a tick rate of 30 Hz means the server updates the game states only thirty times a second. Higher tick rates produce a more responsive game experience, at the cost of increased load on the simulating server.
 
-## Update Rate
+When a server fails to process all incoming client data before the end of the current tick, it produces gameplay issues such as rubber banding, player teleporting, hits getting rejected, and physics failing. As such, there's a balance between having a high tick rate to create responsive gameplay and ensuring that the server has enough time to process all client data before the end of a tick.
 
-What adds an extra delay on top of the travel time of our data (ping), is how often a game sends and receives that data. When a game sends and receives updates at 30Hz (30 updates per second), then there is more time between updates than when it sends and receives updates at 60Hz.
+## Update rate
 
-Update Rates, additional delay:
+Update rate is a measure of how frequently the client sends and receives data to and from the server. It's also measured in hertz and, like tick rate, a higher update rate results in a more responsive game at the cost of increased processing and network demands on the client and server.
 
-<ImageSwitcher 
+<ImageSwitcher
 lightImageSrc="/update-rates-light.png?text=LightMode"
 darkImageSrc="/update-rates-dark.png?text=DarkMode"/>
 
-Low update rates don't only affect the network delay; they also cause issues like "super bullets," where a single hit from a gun deals more damage than it should be able to deal. 
-
-For example, a game server sends 10 updates per second. At this update rate we have 100ms between the updates, which is the same time that we have between two bullets when a gun fires 600 rounds per minute.
+In addition to adding to perceived latency, low update rates can cause their own issues such as 'super bullets', where multiple updates are bundled together and arrive at the same time, causing undesirable behavior.
```

Comment:
SUPER BULLETS.

Reply:
_That's telekinesis, Kyle!_

Bet you haven't thought about Tenacious D in a while.

Reply:
How are the power.... _to move you_. *sick acoustic solo*